build.sh

```diff
@@ -7,19 +7,38 @@ function cleanup() {
     losetup -d "${LOOP_DEVICE_HDD}" || true
     rm -rf "${TMPDIR}"
 }
+
+SSHBUILD=FALSE
+if [ "${1-default}" == "SSH" ]; then
+    SSHBUILD=TRUE
+    # check SSH build dependencies
+    if [ ! -f ./ssh/authorized_keys ]; then
+        echo "You have to create authorized_keys file in ssh folder"
+        exit
+    fi
+    if ! which dropbear; then
+        echo "Please install dropbear: apt install dropbear"
+        exit
+    fi
+fi
+
 trap cleanup EXIT
 
 # Default config for 64-bit Linux and Sedutil
 GRUBSIZE=15 # Reserve this amount of MiB on the image for GRUB (increase this number if needed)
 CACHEDIR="cache"
 TCURL="http://distro.ibiblio.org/tinycorelinux/14.x/x86_64"
-EXTENSIONS="bash.tcz"
 INPUTISO="TinyCorePure64-current.iso"
 OUTPUTIMG="sedunlocksrv-pba.img"
 BOOTARGS="quiet libata.allow_tpm=1"
 SEDUTILURL="https://raw.githubusercontent.com/Drive-Trust-Alliance/exec/master/sedutil_LINUX.tgz"
 SEDUTILBINFILENAME="sedutil-cli"
 SEDUTILPATHINTAR="sedutil/Release_x86_64/${SEDUTILBINFILENAME}"
+if [ $SSHBUILD ]; then
```

Comment:
Are you sure this conditional works properly? To me it looks like it may always evaluate to true... I think you may need to explicitly compare to TRUE.

Reply:
What to say.. Bash:) BTW yes it does work but sure will make it more explicit.

Reply:
done